airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -1373,14 +1381,16 @@ def parse_response(
         stream_slice: Mapping[str, Any] = None,
         next_page_token: Mapping[str, Any] = None,
     ) -> Iterable[Mapping]:
-        for record in response.json().get("jobs"):  # GitHub puts records in an array.
-            yield self.transform(record=record, stream_slice=stream_slice)
+        for record in response.json()["jobs"]:
+            if record.get(self.cursor_field):
```

Comment:
this problems is being fixed [here PR](https://github.com/airbytehq/airbyte/pull/17178), this was caused because the cursor_field `completed_at` can be null, now is `started_at`;